<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSA_UI</class>
 <widget class="QWidget" name="RSA_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="back_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>返回主菜单</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="generate_PQ_layout">
     <item>
      <widget class="QPushButton" name="generate_PQ_button">
       <property name="text">
        <string>生成两个素数p, q</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="generate_PQ_lineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="calculate_N_layout">
     <item>
      <widget class="QPushButton" name="calculate_N_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>计算n</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="calculate_N_lineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="calculate_PHI_layout">
     <item>
      <widget class="QPushButton" name="calculate_PHI_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>计算φ(n)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="calculate_PHI_lineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="generate_E_layout">
     <item>
      <widget class="QPushButton" name="generate_E_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>生成整数e</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="generate_E_lineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="generate_D_layout">
     <item>
      <widget class="QPushButton" name="generate_D_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>计算d</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="generate_D_lineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="generate_M_layout">
     <item>
      <widget class="QPushButton" name="generate_M_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>生成明文</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="generate_M_lineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="encrypt_layout">
     <item>
      <widget class="QPushButton" name="encrypt_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>加密运算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="encrypt_lineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="decrypt_layout">
     <item>
      <widget class="QPushButton" name="decrypt_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>解密运算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="decrypt_lineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
